test/embulk/parser/test_query_string.rb

```diff
@@ -0,0 +1,116 @@
+require "prepare_embulk"
+require "embulk/parser/query-string"
+require "embulk/data_source"
+
+module Embulk
+  module Parser
+    class QueryStringPluginTest < Test::Unit::TestCase
+      class TestParse < self
+        def test_parse_line_without_options
```

Comment:
Class name specifies that these tests check `parse` method already, so method names don't need `parse` term I think.